js/ui/diagram/ui.diagram.floating_panel.js

```diff
@@ -125,6 +125,7 @@ class DiagramFloatingPanel extends DiagramPanel {
             position: this._getPopupPosition(),
             showCloseButton: true,
             copyRootClassesToWrapper: true,
+            hideCopyClassesDeprecationWarning: true,
```

Comment:
Let's name it `_ignoreCopyRootClassesToWrapperDeprecation` by analogy with `_ignoreFunctionValueDeprecation`

```suggestion
            _ignore: true,
```

Reply:
ok